packages/docusaurus-theme-classic/src/theme/Footer/index.js

```diff
@@ -86,7 +87,21 @@ function Footer() {
           <div className="text--center">
             {logo && logo.src && (
               <div className="margin-bottom--sm">
-                <img className="footer__logo" alt={logo.alt} src={logoUrl} />
+                {logo.href ? (
+                  <a
+                    href={logo.href}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className={styles.footerLogoLink}>
+                    <img
+                      className="footer__logo"
+                      alt={logo.alt}
+                      src={logoUrl}
+                    />
+                  </a>
+                ) : (
+                  <img className="footer__logo" alt={logo.alt} src={logoUrl} />
```

Comment:
Shall we extract this out into a component to avoid duplicating it?

Reply:
done :heavy_check_mark: